### PR TITLE
[4.0] Help Links

### PR DESCRIPTION
**Pull request by brianteeman**
Without continuing the debate about what type of links should or should not be on the help page there can be no argument that the glip link doesnt beling here. It is not a help link and more importantly it is not a public link/resource it is a private joomla insiders link that should not be here

image

### CODE REVIEW

File: administrator/components/com_menus/presets/help.xml [hunk 1]
```diff
@@ -160,12 +160,5 @@
 			target="_blank"
 			link="https://github.com/joomla/joomla-cms/"
 		/>
-
-		<menuitem
-			title="MOD_MENU_HELP_GLIP"
```
Comment (Quy) [inline administrator/components/com_menus/presets/help.xml:165]: Remove language string.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 26e048cd212566f591d19104a38578ec5a07f954<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27894">issues.joomla.org/tracker/joomla-cms/27894</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 26e048cd212566f591d19104a38578ec5a07f954<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27894">issues.joomla.org/tracker/joomla-cms/27894</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27894">issues.joomla.org/tracker/joomla-cms/27894</a>.</sub>

**Comment by brianteeman** [comment]
thanks

**Pull request merged**